### DISCUSSION TITLE
Unavailable to import useQuasar while using CLI

### DISCUSSION THREAD

**Discussion by pasjonatprogramowania** [quasarframework/quasar#9885] [General - Components / Directives / etc]
I am getting following error while importing `useQuasar` in this project I am using Quasar CLI I have recommended vue version < 4.0.0 . Someone maybe has idea why this error showing ?

Documentation is not telling that i need to install something more to use `quasarUse`  

https://quasar.dev/vue-composables/use-quasar

In this case, I using this for notify component 
https://quasar.dev/quasar-plugins/notify

![Code_ld6d1UvgKR](https://user-images.githubusercontent.com/44599809/124020387-83199a80-d9ea-11eb-90bd-11699d09d1b1.png)

quasar.conf.js
```
    framework: {
      iconSet: 'material-icons', // Quasar icon set
      lang: 'en-us', // Quasar language pack

      // Possible values for "importStrategy":
      // * 'auto' - (DEFAULT) Auto-import needed Quasar components & directives
      // * 'all'  - Manually specify what to import
      importStrategy: 'auto',

      // For special cases outside of where "auto" importStrategy can have an impact
      // (like functional components as one of the examples),
      // you can manually specify Quasar components/directives to be available everywhere:
      //
      // components: [],
      // directives: [],s

      // Quasar plugins
      plugins: [
        'Notify'
      ],
      config: {
        notify: { /* look at QuasarConfOptions from the API card */ }
      }
    },
```

**Comment by ontwikkelfabriek** [comment]
@pasjonatprogramowania 

Did you fix this already?

The only odd thing I see is that the error message mentions `Quasar` with a capital **Q** instead of `quasar`

Btw you can also use the 'non vue way` to create notifications:
```javascript
import { Notify } from 'quasar'
...
Notify.create({
  message: 'Danger, Will Robinson! Danger!'
})
```

https://quasar.dev/quasar-plugins/notify#basic

**Reply by pasjonatprogramowania** [reply]
No, I can't fix this
But thank you for your suggestion to use 
```
import { Notify } from 'quasar'
...
Notify.create({
  message: 'Danger, Will Robinson! Danger!'
})
```
It works. Thank you so much

**Reply by metalsadman** [reply]
useQuasar only available for QV2 projects.

**Reply by pasjonatprogramowania** [reply]
Yes that's right I create my project with vue 2 thank you so much
